Gemfile

```diff
@@ -38,15 +38,11 @@ gem "omniauth-rails_csrf_protection"
 gem "puma", "~> 6.4.0"
 gem "recaptcha", require: "recaptcha/rails"
 gem "simple_form"
+gem "skylight"
 gem "sprockets-rails"
 gem "stimulus-rails"
 gem "turbo-rails"
 
-group :production do
-  gem "dalli"
```

Comment:
@cdoyle-temple Are you sure this is not needed?  It is part of the memcached integration.

Reply:
Not sure why my reply on another tab doesn't show up here, it's marked pending(?). Anyway
> @sensei100 David and I added them when trying to debug recaptcha. If memcached is still needed, and they are integral to it, then they should stay.

Also, we were not using it before the investigation so I don't think it's necessary.